src/components/Header/Header.styles.tsx

```diff
@@ -8,6 +8,7 @@ import BridgesMenu from '../BridgesMenu'
 import GovernanceMenu from '../GovernanceMenu'
 
 const responsiveTextAndMargin = css`
+margin: 0 12px;
```

Comment:
does this get overridden or nah

Reply:
line 13

Reply:
It's fine, on styled comps using that css snippet, they get 0 12px margin by default, and smaller margins on smaller viewports (with the media queries)

Reply:
ah cool cool
yeah indentation just messing with how I read it then, coolio